Course/Course/Presentation/Outline/CourseOutlineView.swift

```diff
@@ -108,21 +107,19 @@ public struct CourseOutlineView: View {
                                 ? viewModel.courseVideosStructure
                                 : viewModel.courseStructure {
                                 
-                                // MARK: - Sections
-                                if viewModel.config.uiComponents.courseNestedListEnabled {
```

Comment:
@IvanStepanok 
we had logic when `courseNestedListEnabled==true` we skip VerticalsView and go to content directly. Could you please add the same? Thank you